src/main/resources/user/walkthrough/Traversing.md

```diff
@@ -0,0 +1,112 @@
+${HEADER}
+
+${CODE_LINK}
+
+Traversing in Gaffer is a powerful feature, allowing a user to move around a graph, finding relationships between elements.
+
+The main Operations that can be used to traverse a graph are described below using this graph:
+
+```
+
+                 --> 7 <--
+               /           \
+              /             \
+             6  -->  3  -->  4
+ ___        ^         \
+|   |       /          /
+ -> 8 -->  5  <--  2  <
+          ^        ^
+         /        /
+    1 --         /
+     \          /
+       --------
+```
+
+## GetWalks(GetElements, GetElements)
+
+A GetWalks operation is used to retrieve all of the walks in a graph starting from one of a set of provided EntityIds, with a maximum length.
```

Comment:
Change to:

A GetWalks operation is used to retrieve all of the walks in a graph starting from a set of vertices.